finance/portfolio-performance/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        buchen portfolio 0.50.2
+name                portfolio-performance
+categories          finance
+platforms           darwin
+license             EPL-1
+maintainers         @janosch mailbox.org:janosch1 openmaintainer
```

Comment:
You need to put braces around your GitHub handle and email address to link them. 

`maintainers         {@janosch mailbox.org:janosch1} openmaintainer`

Reply:
Good catch. Done.